wagtail-{{ cookiecutter.project_name_kebab }}/pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "wagtail-{{ cookiecutter.project_name_kebab }}"
+authors = [{name = "{{ cookiecutter.full_name }}", email = "{{ cookiecutter.email }}"}]
+description = "{{ cookiecutter.project_short_description }}"
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Intended Audience :: Developers",
+    {% if cookiecutter.open_source_license == 'MIT license' -%}
+    "License :: OSI Approved :: MIT License",
+    {%- elif cookiecutter.open_source_license == 'BSD license' -%}
+    "License :: OSI Approved :: BSD License",
+    {%- elif cookiecutter.open_source_license == 'ISC license' -%}
+    "License :: OSI Approved :: ISC License (ISCL)",
+    {%- elif cookiecutter.open_source_license == 'Apache Software License 2.0' -%}
+    License :: OSI Approved :: Apache Software License",
+    {%- elif cookiecutter.open_source_license == 'GNU General Public License v3' -%}
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    {%- elif cookiecutter.open_source_license == 'Not open source' -%}
+    "License :: Other/Proprietary License",
+    {% endif %}
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Framework :: Django",
+    "Framework :: Django :: 3.0",
+    "Framework :: Django :: 3.1",
+    "Framework :: Django :: 3.2",
+    "Framework :: Wagtail",
+    "Framework :: Wagtail :: 2",
+]
+dynamic = ["version"]
+dependencies = [
+    "Django>=3.0,<4.0",
+    "Wagtail>=2.14,<2.16"
+]
+[project.optional-dependencies]
+testing = [
+    "dj-database-url==0.5.0",
+    "freezegun==0.3.15"
+]
+
+[project.urls]
+Home = "https://github.com/{{ cookiecutter.github_username }}/wagtail-{{ cookiecutter.project_name_kebab }}"
+
+[tool.flit.module]
+name = "wagtail_{{ cookiecutter.project_name_snake }}"
+
+[tool.flit.sdist]
```

Comment:
This replaces `MANIFEST.in`. By default flit includes everything except stuff you've `.gitignore`d
https://flit.pypa.io/en/stable/pyproject_toml.html#sdist-section